### DISCUSSION TITLE
Protecting the master branch

### DISCUSSION THREAD

**Discussion by gillisandrew** [zidoro/pomatez#230] [General]
Hey guys, if were gonna working on this together I think it would be a good idea to stop pushing directly to master (especially force pushes 👀). Repo owner should be able to disable merging without pull request and protect specific branches through repo settings.

Handle updates through pull-requests so contributors can rebase off master and have release-ready code without any broken builds and whatnot.

**Comment by sekwah41** [comment]
Did a force push to fix the commit history as some of the commits didn't properly follow conventional commits. The other part was just a new action though that should have been done as a pr though given its nature I need to do a few to make sure its working fine as you can't replicate that part properly.

Luckily the plugin I've added seems to mostly check that but we need to make 100% sure that's enforced otherwise the features/fixes wont show in the automatic changelogs.
